### PR TITLE
Add support for bash completion.

### PR DESCRIPTION
**Pull request by smimram**

### CODE REVIEW

File: doc/content/install.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ how to use OPAM. In order to use it:
 - [you should have at least OPAM version 2.0](https://opam.ocaml.org/doc/Install.html),
 - you should have at least OCaml version 3.08.0, which can be achieved by typing
   ```
-  opam switch install 3.08.0
+  opam switch create 3.08.0
```
Comment (toots) [inline doc/content/install.md]: `4.08.0`

Comment (toots) [reply doc/content/install.md]: Above as well.

File: configure.ac [hunk 2]
```diff
@@ -127,6 +127,9 @@ if test -z `which pkg-config 2>/dev/null` ; then
   AC_MSG_ERROR(Cannot find pkg-config)
 fi
 
+bashcompletiondir='${prefix}'`pkg-config --variable=completionsdir bash-completion`
+AC_SUBST(bashcompletiondir)
```
Comment (toots) [inline configure.ac]: What's the portability of this? How does it behave when there is no `bash-completion`? Here, I have:
```
% pkg-config --variable=completionsdir bash-completion
Package bash-completion was not found in the pkg-config search path.
Perhaps you should add the directory containing `bash-completion.pc'
to the PKG_CONFIG_PATH environment variable
No package 'bash-completion' found
zsh: exit 1     pkg-config --variable=completionsdir bash-completion
```

File: doc/content/install.md (hunk 1 above)
Comment (smimram) [reply doc/content/install.md]: Just corrected in master.

**Comment by toots** [comment]
This looks good to me now.

File: configure.ac (hunk 2 above)
Comment (smimram) [reply configure.ac]: The new version should behave correctly when the package is not found (a default path is given).

**Pull request merged**